soccer/gameplay/tactics/positions/celebration.py

```diff
@@ -0,0 +1,77 @@
+import behavior
+import robocup
+import constants
+import single_robot_composite_behavior
+import main
+import enum
+import skills
+import random
+import time
+
+class Celebration(single_robot_composite_behavior.SingleRobotCompositeBehavior):
```

Comment:
You don't actually use any subbehaviors so I would just make this a single robot behavior instead of composite